src/components/Licenses/SBOMViewer.tsx

```diff
@@ -84,73 +70,63 @@ const BOMDisplay: React.FC = () => {
           }`}
           onClick={() => setActiveTab("bom")}
         >
-          Care Frontend
+          {t("care_frontend")}
         </button>
         <button
           className={`text-md w-full rounded-md px-4 py-2 transition-all duration-300 md:w-auto ${
             activeTab === "beBom" ? "bg-primary text-white" : "bg-gray-200"
           }`}
           onClick={() => setActiveTab("beBom")}
         >
-          Care Backend
+          {t("care_backend")}
         </button>
       </div>
       <Card className="rounded-lg bg-white p-4 shadow-md transition-all duration-300">
         <div className="mb-4">
           <h2 className="mb-2 text-xl font-semibold text-primary md:text-2xl">
-            {bomData.bomFormat || "N/A"} BOM (Version:{" "}
-            {bomData.version || "N/A"})
+            {t("spdx_sbom_version") + ": " + bomData?.sbom?.spdxVersion ||
+              t("n_a")}
           </h2>
           <p className="text-sm text-gray-500">
-            Created on:{" "}
-            {bomData.metadata?.timestamp
-              ? dayjs(bomData.metadata.timestamp).format("MMMM D, YYYY")
-              : "N/A"}
+            {t("created_on")}{" "}
+            {bomData?.sbom?.creationInfo?.created
+              ? dayjs(bomData.sbom.creationInfo.created).format("MMMM D, YYYY")
+              : t("n_a")}
           </p>
         </div>
         <div className="grid grid-cols-1 gap-4 md:grid-cols-2">
           <h3 className="col-span-full text-lg font-semibold text-primary">
-            Components:
+            {t("packages")}
+            {":"}
           </h3>
-          {bomData.components?.map((component, index) => (
+          {packages.map((pkg: any, index: number) => (
```

Comment:
now that we have strong types defined, let's not use any.

```suggestion
          {packages.map((package, index) => (
```